### PR TITLE
Fix issue with status image being X instead of an hourglass.

### LINKED ISSUE #11: Cache is showing ❌ even though Location && !Warning
Shouldn't this be ⌛ instead?
error

### PR DESCRIPTION
**Pull request by banaag**
Fixes #11

### CODE REVIEW

File: dialog.js [hunk 1]
```diff
@@ -204,7 +204,11 @@ async function setCertDisplayFields(result) {
       document.getElementById('certimg').innerHTML = "✅";
     }
   } else {
-    document.getElementById('certimg').innerHTML = "❌";
+    if (warning == null && location != null) {
```
Comment (twifkak) [inline dialog.js:207]: Invert this if chain, i.e.:

```
if (warning == null && location != null) {
  ⌛
} else if (correctContentType) {
  ✅
} else {
  ❌
}
```

Ditto setDisplayFields below.

Comment (twifkak) [inline dialog.js:207]: (nit) In JS, prefer `===` over `==` and `!==` over `!=` in (almost?) all cases.

**Review by twifkak — APPROVED** [review]

File: dialog.js (hunk 1 above)
Comment (banaag) [reply dialog.js:207]: Done.

Comment (banaag) [reply dialog.js:207]: Done.

**Pull request merged**